docs/source/network/network.md

```diff
@@ -1,1032 +1,165 @@
+
+While it would also be possible for R2 to deploy a new peer to join to channel C2, in this case they have chosen to deploy the P2 to C2. Note that P2 has both the ledger of C1 (called L1) and the ledger of C2 (called L2) on its file system. Similarly, R2 has chosen to modify its application, A2, to be able to be used with C2, while R3's application, A3, is being used with C2.
+
+Logically, this is all very similar to the creation of C1. Two peer organizations come together with an ordering organization to create a channel and join components and a chaincode to it.
+
+Think about this configuration from the standpoint of R2, which is joined to both channels. From their perspective, they might think about both C1 and C2, as well as the components they have joined to both, as the "network", even though both channels are distinct from each other. In this sense, a "network" can also be seen as existing within the perspective of a particular organization as "all of the channels I am a member of and all of the components I own".
+
+Now that we have shown how organizations and their components can be joined to multiple channels, lets talk about how an organization and its components are added to an existing channel.
+
+## Adding an organization to an existing channel
+
+As channels mature, it is natural that its configuration will also mature, reflecting changes in the world that must be reflected in the channel. One of the more common ways a channel will be modified is to add new organizations to it. While it also possible to add more orderer organizations (who may or may not contribute their own nodes), in this example we'll describe the process of how a peer organization, R3, is added to the channel configuration CC1 of channel C1.
+
+**Note that rights and permissions are defined at a channel level. Just because an organization is administrator of one channel does not mean it will be an administrator of a different channel. Each channel is a distinct administrative zone and fully customizable to the use case it's serving.**
+
+![network.7](./network.diagram.7.png)
+
+Although the update to the diagram looks one simple step, adding a new organization to a channel is, at a high level, a three step process:
+
+1. Decide on the new organization's permissions and role. The full scope of these rights must be agreed to before R3 is added to C1 and is beyond the scope of this topic, but comprise the same kinds of questions that must be answered when creating a channel in the first place. What kind of permissions and rights will R3 have on C1? Will it be an admin on the channel, with full rights to sign channel configuration updates of its own? Will its access to any channel resources be restricted (for example, R3 might only be able to write to C1, which means it can propose changes but not sign them)? What chaincodes will R3 install on its peers? What chaincodes will be redefined and recommitted at a channel level to add R3 as an endorser?
```

Comment:
This makes it sound like a single org has admin priv on a channel. By default a majority of orgs must sign off on channel config updates.
Similarly, the default endorsement policy is majority, and if that is the case, the chaincode does not have to be redefined and recommitted upon adding an org.